scripting/include/srccoop/manager.inc

```diff
@@ -546,7 +558,10 @@ public Action Timer_DecrementUntilMapStart(Handle hTimer)
 				if (IsClientInGame(i) && !IsFakeClient(i))
 				{
 					CBasePlayer pClient = CBasePlayer(i);
-					pClient.SetFlags(pClient.GetFlags() & ~FL_ATCONTROLS);
+					// TODO: Look into this.
+					// `FL_FROZEN` is being removed even without the below line in Black Mesa (i doubt this affects other games).
```

Comment:
If you let the BM timer run to 0.00 it will do its deathmatch post-warmup start procedure. That's why I offset it by 1. Nowadays the effects are mostly mitigated because the intermission length is defaulted to 0 in base config and there are gamerules hook preventing entity respawns etc.

Reply:
Oh yeah that is what I assumed. I just put that comment down just incase we run into issues in the future.